app/assets/images/payment_icons/uob.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="38" height="24" viewBox="0 0 38 24" role="img" aria-labelledby="pi-uob"><title id="pi-uob">UOB</title>><path opacity=".1" d="M35 0H3a3 3 0 0 0-3 3v18a3 3 0 0 0 3 3h32a3 3 0 0 0 3-3V3a3 3 0 0 0-3-3z"/><path fill="#FFF" d="M35 1a2 2 0 0 1 2 2v18a2 2 0 0 1-2 2H3a2 2 0 0 1-2-2V3c0-1.1.9-2 2-2h32"/><path fill="#D90B14" d="M8.6 9.5V11c0 .3 0 .4.4.4.2 0 .3-.1.3-.4V7.9c0-.4.1-.6.6-.6.4 0 .5.2.5.6v2.8c0 .8-.1.8.8.7.4 0 .6.1.6.6 0 .4-.1.5-.6.5-.8 0-.8 0-.8 1V16c0 .4 0 .6-.5.6s-.6-.2-.6-.6V13c0-.3 0-.5-.3-.5-.4 0-.4.2-.4.5v2.8c0 .4.1 1-.6.9-.7 0-.4-.6-.5-1V13c0-.3 0-.5-.4-.5s-.4.3-.4.6v3c0 .4 0 .6-.5.6s-.5-.2-.5-.6V13c0-.3 0-.5-.4-.5-.3 0-.4.2-.4.5v3c0 .4-.1.7-.5.7-.5 0-.6-.3-.6-.6V13c0-.4 0-.5-.4-.5a3 3 0 0 1-.6 0c-.3 0-.4-.3-.4-.6 0-.2 0-.4.4-.5.3 0 .8.2 1 0v-1.1-2.5c0-.3.2-.5.5-.5.4 0 .6.2.6.6v3c0 .4 0 .6.4.6s.4-.2.4-.5V7.9c0-.4 0-.6.5-.6.4 0 .5.2.5.6V11c0 .3 0 .5.4.5.3 0 .4-.1.4-.5V7.8c0-.4.2-.5.5-.5.4 0 .6.1.6.5v1.7"/><path fill="#0C1858" d="M29.1 12V8.7c0-.4.1-.6.5-.6h4c.7.1 1.2.5 1.4 1.2.2.8.3 1.6-.5 2.2-.2.1-.3.3 0 .4 1 .3 1.2 1.5 1 2.3-.2 1-.8 1.6-1.8 1.6h-3c-1.6 0-1.6 0-1.6-1.5V12m2.9 2.8h.8c.8 0 1.1-.4 1.1-1.2 0-.7-.4-1.1-1-1.2h-1c-1.1 0-1.1 0-1.1 1.2s0 1.2 1.2 1.2m0-5.6h-.8c-.3 0-.4 0-.4.4 0 .6-.2 1.3.1 1.6.3.4 1 0 1.5.1h.1c.7 0 1-.4 1-1.1 0-.7-.2-1-1-1H32M24.7 8c1.5 0 2.7.4 3.3 1.9.3.8.3 1.7.2 2.7 0 .8-.1 1.5-.6 2.2-1 1.1-2.3 1.3-3.6 1.1-2.7-.3-3-2.7-2.9-4.8 0-.6.2-1.3.7-2 .7-1 1.8-1 2.9-1.1m2 4c0-.5 0-1-.2-1.6-.3-1-.9-1.3-2.1-1.2-.9 0-1.5.6-1.6 1.4v2.7c.1.7.7 1.3 1.3 1.5 1 .2 1.7-.1 2.2-.8.4-.7.3-1.3.3-2M20.2 11.2v2.6a2 2 0 0 1-1.7 2c-1.2.2-2.5.3-3.6-.3-.7-.4-1-1-1-1.7V8.6c0-.3 0-.4.2-.5.9-.3 1.4 0 1.4 1v4.2c0 1 .3 1.3 1.3 1.3h.9c.6-.2.8-.5.8-1.1V8.8c0-.6.3-.8.9-.8.6 0 .8.2.8.8v2.4z"/><path fill="#F3F3F3" d="M32 14.8c-1.2 0-1.2 0-1.2-1.2s0-1.2 1.2-1.2h.8c.7 0 1.1.5 1.1 1.2 0 .8-.3 1.1-1 1.2h-1M32 9.2h.5c.8 0 1 .3 1 1s-.3 1-1 1.1c-.6 0-1.3.3-1.6-.1-.3-.3 0-1-.1-1.6 0-.3.1-.4.4-.4h.8M26.6 12c0 .7 0 1.3-.3 2-.5.7-1.2 1-2.2.8-.6-.2-1.2-.8-1.3-1.5v-2.7c.1-.8.7-1.4 1.6-1.4 1.2 0 1.8.3 2.1 1.2l.1 1.6"/></svg>
```

Comment:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="38" height="24" viewBox="0 0 38 24" role="img" aria-labelledby="pi-uob"><title id="pi-uob">UOB</title><path opacity=".1" d="M35 0H3a3 3 0 0 0-3 3v18a3 3 0 0 0 3 3h32a3 3 0 0 0 3-3V3a3 3 0 0 0-3-3z"/><path fill="#FFF" d="M35 1a2 2 0 0 1 2 2v18a2 2 0 0 1-2 2H3a2 2 0 0 1-2-2V3c0-1.1.9-2 2-2h32"/><path fill="#D90B14" d="M8.6 9.5V11c0 .3 0 .4.4.4.2 0 .3-.1.3-.4V7.9c0-.4.1-.6.6-.6.4 0 .5.2.5.6v2.8c0 .8-.1.8.8.7.4 0 .6.1.6.6 0 .4-.1.5-.6.5-.8 0-.8 0-.8 1V16c0 .4 0 .6-.5.6s-.6-.2-.6-.6V13c0-.3 0-.5-.3-.5-.4 0-.4.2-.4.5v2.8c0 .4.1 1-.6.9-.7 0-.4-.6-.5-1V13c0-.3 0-.5-.4-.5s-.4.3-.4.6v3c0 .4 0 .6-.5.6s-.5-.2-.5-.6V13c0-.3 0-.5-.4-.5-.3 0-.4.2-.4.5v3c0 .4-.1.7-.5.7-.5 0-.6-.3-.6-.6V13c0-.4 0-.5-.4-.5a3 3 0 0 1-.6 0c-.3 0-.4-.3-.4-.6 0-.2 0-.4.4-.5.3 0 .8.2 1 0v-1.1-2.5c0-.3.2-.5.5-.5.4 0 .6.2.6.6v3c0 .4 0 .6.4.6s.4-.2.4-.5V7.9c0-.4 0-.6.5-.6.4 0 .5.2.5.6V11c0 .3 0 .5.4.5.3 0 .4-.1.4-.5V7.8c0-.4.2-.5.5-.5.4 0 .6.1.6.5v1.7"/><path fill="#0C1858" d="M29.1 12V8.7c0-.4.1-.6.5-.6h4c.7.1 1.2.5 1.4 1.2.2.8.3 1.6-.5 2.2-.2.1-.3.3 0 .4 1 .3 1.2 1.5 1 2.3-.2 1-.8 1.6-1.8 1.6h-3c-1.6 0-1.6 0-1.6-1.5V12m2.9 2.8h.8c.8 0 1.1-.4 1.1-1.2 0-.7-.4-1.1-1-1.2h-1c-1.1 0-1.1 0-1.1 1.2s0 1.2 1.2 1.2m0-5.6h-.8c-.3 0-.4 0-.4.4 0 .6-.2 1.3.1 1.6.3.4 1 0 1.5.1h.1c.7 0 1-.4 1-1.1 0-.7-.2-1-1-1H32M24.7 8c1.5 0 2.7.4 3.3 1.9.3.8.3 1.7.2 2.7 0 .8-.1 1.5-.6 2.2-1 1.1-2.3 1.3-3.6 1.1-2.7-.3-3-2.7-2.9-4.8 0-.6.2-1.3.7-2 .7-1 1.8-1 2.9-1.1m2 4c0-.5 0-1-.2-1.6-.3-1-.9-1.3-2.1-1.2-.9 0-1.5.6-1.6 1.4v2.7c.1.7.7 1.3 1.3 1.5 1 .2 1.7-.1 2.2-.8.4-.7.3-1.3.3-2M20.2 11.2v2.6a2 2 0 0 1-1.7 2c-1.2.2-2.5.3-3.6-.3-.7-.4-1-1-1-1.7V8.6c0-.3 0-.4.2-.5.9-.3 1.4 0 1.4 1v4.2c0 1 .3 1.3 1.3 1.3h.9c.6-.2.8-.5.8-1.1V8.8c0-.6.3-.8.9-.8.6 0 .8.2.8.8v2.4z"/><path fill="#F3F3F3" d="M32 14.8c-1.2 0-1.2 0-1.2-1.2s0-1.2 1.2-1.2h.8c.7 0 1.1.5 1.1 1.2 0 .8-.3 1.1-1 1.2h-1M32 9.2h.5c.8 0 1 .3 1 1s-.3 1-1 1.1c-.6 0-1.3.3-1.6-.1-.3-.3 0-1-.1-1.6 0-.3.1-.4.4-.4h.8M26.6 12c0 .7 0 1.3-.3 2-.5.7-1.2 1-2.2.8-.6-.2-1.2-.8-1.3-1.5v-2.7c.1-.8.7-1.4 1.6-1.4 1.2 0 1.8.3 2.1 1.2l.1 1.6"/></svg>
```

There's an extra chevron on `<title>`; currently it says `<title id="pi-uob">UOB</title>>`. This suggestion fixes that.